src/bundles/files/utils.js

```diff
@@ -19,7 +26,8 @@ export const make = (basename, action, options = {}) => (...args) => async (args
   try {
     data = await action(getIpfs(), ...args, id, args2)
 
-    const paths = args[0] ? args[0].flat() : []
+    // TODO: Add a comment explaining what is going on here.
```

Comment:
Don't forget this TODO 😄

Reply:
I was actually hoping you or someone with more knowledge of the code base could do that, because I really have no clue what is going on here.

Reply:
Submitted a followup https://github.com/ipfs-shipyard/ipfs-webui/issues/1578